src/Slack/MessageSender.php

```diff
@@ -140,13 +145,20 @@ public function sendSecretMessage(SecretSanta $secretSanta, string $giver, strin
             ],
         ];
 
+        $messageParameters = [
+            'channel' => $giver,
+            'text' => $fallbackText,
+            'blocks' => json_encode($blocks),
+        ];
+
         try {
-            $response = $this->clientFactory->getClientForToken($token)->chatPostMessage([
-                'channel' => $giver,
-                'icon_url' => 'https://secret-santa.team/images/logo.png',
-                'text' => $fallbackText,
-                'blocks' => json_encode($blocks),
-            ]);
+            if ($schedule && !$isSample) {
+                $messageParameters['post_at'] = (int) $schedule;
+                $response = $this->clientFactory->getClientForToken($token)->chatScheduleMessage($messageParameters);
+            } else {
+                $messageParameters['icon_url'] = 'https://secret-santa.team/images/logo.png';
```

Comment:
There is no "icon_url" on scheduled messages? (did you check the rendering?)

Reply:
The "icon_url" parameter doesn't exist for the chatScheduleMessage method, I didn't find a way to add it.

Reply:
Did you check the rendering in case of scheduled message? Ie, what is displayed in bot avatar?

Reply:
I actually removed it because, as it turns out, it wasn't even needed. The icon is displayed without the parameter, it must've been enforced somewhere else in the config